pkg/vm/engine/disttae/txn_table.go

```diff
@@ -1053,10 +1069,13 @@ func (tbl *txnTable) tryFastRanges(
 		}
 		objDataMeta = objMeta.MustDataMeta()
 		cnt += objDataMeta.BlockCount()
-		pkZM := objDataMeta.MustGetColumn(uint16(tbl.primaryIdx)).ZoneMap()
-		if !pkZM.ContainsKey(val) {
-			continue
+		if obj.ZMIsEmpty() {
```

Comment:
it will not check obj.ZMIsEmpty case.
We don't need to change here, since we had fetched the metadata before